utopia-remix/app/routes/v1.project.$id.owner.tsx

```diff
@@ -1,10 +1,11 @@
 import { LoaderFunctionArgs } from '@remix-run/node'
 import { proxy } from '../util/proxy.server'
 import { handle, handleOptions } from '../util/api.server'
+import { ALLOW } from '../handlers/validators'
 
 export async function loader(args: LoaderFunctionArgs) {
   return handle(args, {
     OPTIONS: handleOptions,
-    GET: proxy,
+    GET: { handler: proxy, validator: ALLOW },
```

Comment:
this should also validate the project access

Reply:
fixed